### DISCUSSION TITLE
What is the best way to localize emails?

### DISCUSSION THREAD

**Discussion by michalvadak** [maizzle/framework#812] [Q&A]
Hello,

I was wondering what is the best way to localize my email templates. It would be great to have just one file that will be able to build into multiple files based on localization. Is this possible?

Thank you!

**Comment by cossssmin** [comment]
Hey, so right now it isn’t included in Maizzle, but if you wanted you could integrate something like i18next and maybe create a function (like in the `beforeCreate` event) that takes a string and returns the translation.

This is on the list for a future 4.x feature release, though need to test thoroughly and make sure we don’t introduce any breaking changes (that would require a major 5.0 release).

**Comment by bojangles-m** [comment]
Hi, 

is it possible to build multiple files with the command `maizzle build production` or in it needs own render method which loops over all languages and compiles templates in destination folder?

Thnx!
